me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/IDmeWebVerify.java

```diff
@@ -82,12 +83,28 @@ private void checkInitialization() {
    * @param listener The listener that will be called when the login process is finished.
    */
   public void login(@NonNull Activity activity, @NonNull IDmeScope scope, @NonNull IDmeGetAccessTokenListener listener) {
+    login(activity, scope, null, listener);
+  }
+
+  /**
+   * Starts the login process
```

Comment:
From iOS SDK doc: 

```
This function should be used if it is known if the user wants to sign in or sign up. Otherwise works the same as verifyUserInViewController:scope:webVerificationResults
```

Maybe you can add it.